cli.js

```diff
@@ -101,5 +101,5 @@ Promise.resolve()
     }
   })
   .then(() => {
-    return tasks.run();
```

Comment:
Are there any errors which would surface here separate from the cases where there is duplication?

Reply:
I don't think it's possible for that to happen. Within `listr`, any [caught errors](https://github.com/SamVerschueren/listr/blob/d28fb95d192fc06b8489ee11450ee29e820ee16f/lib/task.js#L175-L199) are [reported](https://github.com/SamVerschueren/listr/blob/d28fb95d192fc06b8489ee11450ee29e820ee16f/lib/task.js#L193) and then [rethrown](https://github.com/SamVerschueren/listr/blob/d28fb95d192fc06b8489ee11450ee29e820ee16f/lib/task.js#L197).

Anything [outside the task list](https://github.com/bugsnag/bugsnag-sourcemaps/blob/e62d065cec286191af21b6c2190f69a6c70329a7/cli.js#L89-L102) promise chain with be unhandled and logged to the console.